_blueprints/remote_issue_trackers.adoc

```diff
@@ -326,3 +326,325 @@ https://github.com/plish/Trolly
+-----
+
+[[auth-models]]
+*Auth models:*
+
+1. There are 2 authentication models:
+    * Basic authentication
+	* Two factor authentication
+
+2. Basic Authentication:
+	* send a HTTP GET request with proper credentials to the endpoint:
+		+ https://bugzilla.mozilla.org/rest/login?login=username&password=password
+	* It returns an id and token
+	* Use token in URL.
+		+ https://bugzilla.mozilla.org/rest/bug?token=272345-L1KydUNCwq&f1=cf_tracking_firefox31&o1=equals&v1=%2B&include_fields=id
+
+2. Two factor authenticatoon:
+	* If you have two-factor authentication on your bugzilla account, then create a new API key and pass it to each request
+		+ requests.get('https://bugzilla.mozilla.org/rest/bug', params={'id': '1214433', 'api_key': 'xyz'})
+
+3. Cookie authentication is not accepted.
+
+[[auth-levels]]
+*Authorization Level*
+
+1. Permission to do stuff in Bugzilla are defined by groups and Bugzilla users get membership to certain groups manually or automatically
+
+2. Groups:
+	* restrict visibility of bugs to a specific set of users
+	* Types:
+		 - Generic group
+		 - Product based group
+
+|================================================================
+|*Administrators*| *Permissions* 
+| who have "editbugs" priviledges | can edit all bug fields 
+| who have "editkeywords" priviledges | can create, destroy, edit keywords
+| who have "editusers" priviledges | can create, update, block user profiles
+| who have "canconfirm" priviledges	| can confirm a bug or mark it as duplicate
+| who have "admin" priviledges | can edit fields, groups -> total control over Bugzilla
+|================================================================
+
+[[bugzilla-golang-libraries]]
+Bugzilla Golang libraries
+~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Could you change headers to use the short format: === Header. 

Check level used (number of =) further up in the document.